BootloaderCorePkg/Library/FspApiLib/FspVariableServices.c

```diff
@@ -0,0 +1,127 @@
+/** @file
+
+  Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+#include <Uefi.h>
+#include <FspApiLibInternal.h>
+#include <Library/BootloaderCoreLib.h>
+#include <Library/FspApiLib.h>
+#include <Library/VariableLib.h>
+
+/**
+  This function handles variable services requests from FspMemoryInit and FspSiliconInit.
+  This should be called immediately after returning from the respective FSP entrypoint.
+
+  @param[in] FspStatus            The status returned by the FSP entry point.
+  @param[in] MultiPhaseFunction   The Multiphase function for this FSP phase (Mem vs Si)
+
+  @retval Status
+ */
+EFI_STATUS
+EFIAPI
+FspVariableHandler(
+  IN EFI_STATUS                FspStatus,
+  IN FSP_MULTI_PHASE_FUNCTION  MultiPhaseFunction
+  )
+{
+  EFI_STATUS                                    Status;
+  FSP_MULTI_PHASE_PARAMS                        MultiPhaseInitParams;
+  FSP_MULTI_PHASE_VARIABLE_REQUEST_INFO_PARAMS  *FspVarReqParams;
+
+  Status = EFI_SUCCESS;
+
+  while (FspStatus == FSP_STATUS_VARIABLE_REQUEST) {
+    DEBUG((DEBUG_VERBOSE, "FspVariableHandler: Got FSP_STATUS_VARIABLE_REQUEST\n"));
+    // Request details of variable request from Multi-phase API.
+    MultiPhaseInitParams.MultiPhaseAction = EnumMultiPhaseGetVariableRequestInfo;
+    MultiPhaseInitParams.PhaseIndex = 0;
+    Status = MultiPhaseFunction(&MultiPhaseInitParams);
+    ASSERT_EFI_ERROR(Status);
+    if (EFI_ERROR(Status)) {
+      break;
+    }
+
+    FspVarReqParams = (FSP_MULTI_PHASE_VARIABLE_REQUEST_INFO_PARAMS*)MultiPhaseInitParams.MultiPhaseParamPtr;
+    ASSERT (FspVarReqParams != NULL);
+    if (FspVarReqParams == NULL) {
+      Status = EFI_UNSUPPORTED;
+      break;
+    }
+
+    DEBUG((DEBUG_VERBOSE, "FSP Variable Request %d\n", FspVarReqParams->VariableRequest));
+    if (NULL != FspVarReqParams->VariableName) {
+      DEBUG((DEBUG_VERBOSE, "\tVariableName %s\n", FspVarReqParams->VariableName));
+    }
+    if (NULL != FspVarReqParams->VariableGuid) {
+      DEBUG((DEBUG_VERBOSE, "\tVariableGuid %g\n", FspVarReqParams->VariableGuid));
+    }
+    if (NULL != FspVarReqParams->DataSize) {
+      DEBUG((DEBUG_VERBOSE, "\tDataSize 0x%X\n", *(FspVarReqParams->DataSize)));
+    }
+    DEBUG((DEBUG_VERBOSE, "\tData ptr 0x%X\n", FspVarReqParams->Data));
+
+    switch (FspVarReqParams->VariableRequest) {
+      case EnumFspVariableRequestGetVariable:
+        Status = GetVariable(
+          FspVarReqParams->VariableName,
```

Comment:
Subsequent lines of multi-line function calls should line up two spaces from the beginning of the function name. In this case the function name is GetVariable. So it should be changed as below (same to other functions)..
        Status = GetVariable (
                        FspVarReqParams->VariableName,